ci/ray_ci/container.py

```diff
@@ -37,6 +37,7 @@
     "BUILDKITE_LABEL",
     "BUILDKITE_BAZEL_CACHE_URL",
     "BUILDKITE_PIPELINE_ID",
+    "BUILDKITE_PULL_REQUEST",
```

Comment:
the wheel build process uses this variable to skip uploading cache to bazel on pr; this help reduces the level of noises in the log